### PR TITLE
Fix three typos in Chills copy

### PR DESCRIPTION
**Pull request by tpaik18**
The "Achievment" typo was not from the copy doc. 
The other two typos are from the copy doc, and I have made suggested edits in the copy doc to match.

### CODE REVIEW

File: FluStudy_us/src/i18n/locales/en.json [hunk 1]
```diff
@@ -811,7 +811,7 @@
   },
   "MucusUTM": {
     "title": "Collect sample from nose",
-    "desc": "Now you'll swab your nose the same was you did in the first test.",
+    "desc": "Now you'll swab your nose the same as you did in the first test.",
```
Comment (smeds1) [inline FluStudy_us/src/i18n/locales/en.json]: I think this was supposed to be "way"

File: FluStudy_us/src/i18n/locales/en.json [hunk 2]
```diff
@@ -841,7 +841,7 @@
   },
   "WebSiteForPickup": {
     "title": "Schedule a FedEx pickup online",
-    "desc": "You can fill out a form on the FedEx website to arrange for a pickup. If you leave the package on your doorstep for pickup, try not leave it in direct sunlight or freezing temperatures.\n\nWhen filling out the form on the FedEx site, please note:",
+    "desc": "You can fill out a form on the FedEx website to arrange for a pickup. If you leave the package on your doorstep for pickup, try to not leave it in direct sunlight or freezing temperatures.\n\nWhen filling out the form on the FedEx site, please note:",
```
Comment (smeds1) [inline FluStudy_us/src/i18n/locales/en.json]: I like "not to" better than "to not"

File: FluStudy_us/src/i18n/locales/en.json (hunk 1 above)
Comment (tpaik18) [reply FluStudy_us/src/i18n/locales/en.json]: @smeds1 Oh!

File: FluStudy_us/src/i18n/locales/en.json (hunk 2 above)
Comment (tpaik18) [reply FluStudy_us/src/i18n/locales/en.json]: @smeds1 Me too, and I first did "not to" but then I saw it was "to not" in the other screen that has the same sentence.

**Comment by tpaik18** [comment]
Shawna just approved some of my edits on a few other screens so I am rolling those fixes into this PR.

**Pull request merged**